lib/liquid/combinators/tags/if.ex

```diff
@@ -28,8 +28,32 @@ defmodule Liquid.Combinators.Tags.If do
     |> optional(parsec(:__parse__))
   end
 
+  @type unless_tag :: [
+          unless: [
+            conditions:
+              General.condition()
+              | [logical: [or: General.condition()]]
+              | [logical: [and: General.condition()]],
+            body: Liquid.t(),
+            elsif: If.t(),
```

Comment:
are you sure `elseif` returns a `if` type? Maybe it only returns the internal type of `if` (which wll be reused by `unless`)